src/Microsoft.Azure.WebJobs.Extensions.OpenApi.CLI/Service/Interface/ICustomApiMockCreator.cs

```diff
@@ -0,0 +1,9 @@
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.CLI.Model;
+
+namespace Microsoft.Azure.WebJobs.Extensions.OpenApi.CLI.Service.Interface
```

Comment:
Can we change the folder structure from `Service.Interface` to `Abstractions` and also change the namespace accordingly?

Reply:
Changed to match request in new commit